### PR TITLE
TELCODOCS:1103c - Update to kataconfig file in 4.10

### PR DESCRIPTION
**Pull request by MirzWeiss**
Version(s): 4.10

Issue: [TELCODOCS-1103](https://issues.redhat.com/browse/TELCODOCS-1103)

[Link to docs preview](https://55841--docspreview.netlify.app/openshift-enterprise/latest/sandboxed_containers/deploying-sandboxed-container-workloads.html#sandboxed-containers-create-kataconfig-resource-web-console_deploying-sandboxed-containers)

QE review:
- [x] QE has approved this change.

### CODE REVIEW

File: modules/sandboxed-containers-create-kataconfig-resource-web-console.adoc [hunk 1]
```diff
@@ -66,8 +66,9 @@ spec:
   kataMonitorImage: registry.redhat.io/openshift-sandboxed-containers/osc-monitor-rhel8:1.2.0
   kataConfigPoolSelector:
     matchLabels:
-      <label_key>: '<label_value>'
+      <label_key>: '<label_value>' <1>
 ----
+<1> Labels in `kataConfigPoolSelector` only support single values; `nodeSelector` syntax is not supported.----
```
Comment (eohartman) [inline modules/sandboxed-containers-create-kataconfig-resource-web-console.adoc:71]: After the sentence, the "----" shows up in the preview. Can you please remove that?

**Review by eohartman — CHANGES_REQUESTED** [review]

**Comment by eohartman** [comment]
Looks great except for one tiny change.

File: modules/sandboxed-containers-create-kataconfig-resource-web-console.adoc [hunk 2]
```diff
@@ -66,8 +66,9 @@ spec:
   kataMonitorImage: registry.redhat.io/openshift-sandboxed-containers/osc-monitor-rhel8:1.2.0
   kataConfigPoolSelector:
     matchLabels:
-      <label_key>: '<label_value>'
+      <label_key>: '<label_value>' <1>
 ----
+<1> Labels in `kataConfigPoolSelector` only support single values; `nodeSelector` syntax is not supported.
```
Comment (opayne1) [inline modules/sandboxed-containers-create-kataconfig-resource-web-console.adoc:71]: This renders strange. I wonder if it is picking up the ---- from line 70 maybe?

Screenshot 2023-02-23 at 7 38 29 AM

**Review by opayne1 — COMMENTED** [review]
Just a small nit I noticed from the preview. If you want, you can ping me directly and I can merge once it is fixed. Thanks!

File: modules/sandboxed-containers-create-kataconfig-resource-web-console.adoc (hunk 2 above)
Comment (MirzWeiss) [reply modules/sandboxed-containers-create-kataconfig-resource-web-console.adoc:71]: @opayne1  Figured out the issue. The preview didn't update automatically, and this was a mistake in a previous version that was fixed during peer review.  Manually generating an updated preview now.

**Pull request merged**